### DISCUSSION TITLE
Does community edition supports adding site via api?

### DISCUSSION THREAD

**Discussion by ahsanbod** [plausible/analytics#4329] [Self-Hosted Support] [answered]
I am getting 404 while using the endpoint /api/v1/sites

**Comment by ruslandoga** [comment, marked as answer]
👋 @ahsanbod 

Sites API is not included in CE.

**Reply by ahsanbod** [reply]
Can we pull site stats without adding it to the dashboard then?

**Reply by ruslandoga** [reply]
I don't think events would be saved if the site they are for is not registered. So, no.

**Comment by pvdptje** [comment]
Are there any plans of adding this? This is actually one of the reasons I went with plausible only to find out it wasn't possible.

**Reply by ruslandoga** [reply]
👋 @pvdptje 

AFAIK there are no plans right now to add Sites API to Plausible CE.

**Comment by genesis-solution** [comment]
Hi @ruslandoga 

Is there no way to add a site using API in community edition?

**Reply by ruslandoga** [reply]
👋 @genesis-solution 

AFAIK no.

**Comment by genesis-solution** [comment]
Thank you for your quick response, @ruslandoga 👍
